pyproject.toml

```diff
@@ -43,6 +43,7 @@ dependencies = [
 llfuse = ["llfuse >= 1.3.8"]
 pyfuse3 = ["pyfuse3 >= 3.1.1"]
 nofuse = []
+s3 = ["borgstore[s3]"]
```

Comment:
did you try if that works? borgstore is already specified in line 34.

also, the version requirement is missing here.

Reply:
It does work as it's installing optional its optional dependencies. I had assumed the main dependencies already had the version restraint, but I will define the version here as well.